README.md

```diff
@@ -2,17 +2,24 @@
 
 Source code repo for the pelux.io website
 
-## How to build
+## How to build the web site locally
 
-This website uses [jekyll](https://jekyllrb.com/) to build static HTML. These are the steps to build the web site locally:
-*NB: You'll want to have jekyll and bundler installed on your system first.*
+This website uses [jekyll](https://jekyllrb.com/) to build static
+HTML. You'll need some dependencies to build the site locally. These
+dependencies are best to pull from upstream since they rely on
+versions that are known to work together. Install the gems jekyll
+bundler and minima this way;
+
+    sudo gem install jekyll bundler minima
```

Comment:
See the bug report -- the binary (/usr/local/bin/bundle) needs to be installed *before* you call it. :-)